docs/deployment/plugins/k8s/index.rst

```diff
@@ -751,6 +830,122 @@ Specify plugin configuration
                       sidecar: sidecar
                       container_array: k8s-array
                       spark: spark
+
+        .. group-tab:: Flyte sandbox
+
+          Create a file named ``values-override.yaml`` and add the following config to it:
+
+          .. note::
+
+            Within the flyte-binary block, the value of inline.storage.signedURL.stowConfigOverride.endpoint should be set to the corresponding node Hostname/IP on the MinIO pod if you are deploying on a Kubernetes cluster.
```

Comment:
Is this particular to `flyte-core`? I haven't had to use it with `flyte-binary` on-prem

Reply:
And it is possible that the two pods are not on the same K8S node, so the settings in this [line](https://github.com/flyteorg/flyte/blob/master/charts/flyte-sandbox/values.yaml#L53) may need to be set. Node IP

![Screenshot 2024-01-16 at 11 25 35 PM](https://github.com/flyteorg/flyte/assets/14800485/c59b6d82-0afa-48e9-867a-bf07ed7cb113)

![Screenshot 2024-01-16 at 11 18 32 PM](https://github.com/flyteorg/flyte/assets/14800485/710c8d92-ef90-4b0b-af25-6e135b609f65)

Reply:
by the way, Minio service uses NodePort
https://github.com/flyteorg/flyte/blob/master/charts/flyte-sandbox/values.yaml#L116-119